math/src/main/java/smile/math/matrix/CholeskyDecomposition.java

```diff
@@ -55,10 +55,17 @@
     private double[][] L;
 
     /**
+     * Symmetric and positive definite flag.
+     * 
+     * @serial is symmetric and positive definite flag.
+     */
+    private final boolean isSPD;
+    
+    /**
      * Constructor. Used by newInstance().
      */
     private CholeskyDecomposition() {
-        
+        isSPD = true;
```

Comment:
Is this correct behavior? Note this constructor is used by newInstance(), which are used by QR and SVD.